Description: Change the packaging type since the bundle plugin is disabled
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/fontbox/pom.xml
+++ b/fontbox/pom.xml
@@ -26,7 +26,7 @@
   </parent>
 
   <artifactId>fontbox</artifactId>
-  <packaging>bundle</packaging>
+  <packaging>jar</packaging>
 
   <name>Apache FontBox</name>
   <description>
--- a/jempbox/pom.xml
+++ b/jempbox/pom.xml
@@ -26,7 +26,7 @@
   </parent>
 
   <artifactId>jempbox</artifactId>
-  <packaging>bundle</packaging>
+  <packaging>jar</packaging>
 
   <name>Apache JempBox</name>
   <description>
--- a/pdfbox/pom.xml
+++ b/pdfbox/pom.xml
@@ -28,7 +28,7 @@
     </parent>
 
     <artifactId>pdfbox</artifactId>
-    <packaging>bundle</packaging>
+    <packaging>jar</packaging>
 
     <name>Apache PDFBox</name>
     <description>
